docs/how_to/install.rst

```diff
@@ -68,6 +68,11 @@ Other Plugins
 * djangocms-oembed
 * djangocms-table
 * djangocms-flash
+* djangocms-googlemap
+* djangocms-inherit
+* djangocms-picture
+* djangocms-teaser
+* djangocms-video
```

Comment:
This is just a suggested list, with some examples. I don't see the need to add more than a minimum. The user will decide what they need.